core/src/main/scala/org/apache/spark/deploy/FaultToleranceTest.scala

```diff
@@ -113,7 +112,7 @@ private object FaultToleranceTest extends App with Logging {
     assertValidClusterState()
 
     killLeader()
-    delay(30 seconds)
+    delay(30.seconds)
```

Comment:
Personally I prefer `30 seconds`. I counted 3 non-time-related usages in your change, so this feels like a lot of churn for not much gain - we could just fix those and remove the unnecessary imports where postfix ops are not used.

It'd also help if it were possible to turn these warnings into errors, since people don't generally pay attention to warnings...